D:\Documentos\Github\ZLEL proiektua\zlel\zlel_p1.py:514: RuntimeWarning: invalid value encountered in double_scalars
  alphaR = (Ies / Ics) * alphaF
0.6
wololo
-0.0 nan 0.0 -0.0 nan 0.0
0.6
wololo
-0.0 nan 0.0 -0.0 nan 0.0
0.6
wololo
-0.0 nan 0.0 -0.0 nan 0.0
0.6
wololo
-0.0 nan 0.0 -0.0 nan 0.0
Traceback (most recent call last):
  File "zlel_main.py", line 25, in <module>
    zl1.solveCircuit(filename)
  File "D:\Documentos\Github\ZLEL proiektua\zlel\zlel_p1.py", line 1017, in solveCircuit
    M = get_M_matrix(branch_list)
  File "D:\Documentos\Github\ZLEL proiektua\zlel\zlel_p1.py", line 545, in get_M_matrix
    matrix[i] = branch_list[i].get_M_row(branch_list)
  File "D:\Documentos\Github\ZLEL proiektua\zlel\zlel_p1.py", line 354, in get_M_row
    row[get_branch_by_name(self.control,
AttributeError: 'NoneType' object has no attribute 'N'
